docs/define_nodes.rst

```diff
@@ -0,0 +1,79 @@
+Create a full schema
+=======================
+While that setup was certainly quick and easy, it is not very useful yet. Next you will want to create a graph-like schema that maps to your business logic so you can query against it.
```

Comment:
Each of the docs pages should ideally stand on its own - they might come directly here instead of going the Getting Started first.